packages/amplify-category-notifications/src/channel-email.ts

```diff
@@ -61,27 +50,11 @@ export const enable = async (context:$TSContext, successMessage: string|undefine
     if (context.exeInfo.serviceMeta.output[channelName]) {
       channelOutput = context.exeInfo.serviceMeta.output[channelName];
     }
-    const questions = [
-      {
-        name: 'FromAddress',
-        type: 'input',
-        message: "The 'From' Email address used to send emails",
-        default: channelOutput.FromAddress,
-      },
-      {
-        name: 'Identity',
-        type: 'input',
-        message: 'The ARN of an identity verified with SES',
-        default: channelOutput.Identity,
-      },
-      {
-        name: 'RoleArn',
-        type: 'input',
-        message: "The ARN of an IAM Role used to submit events to Mobile notifications' event ingestion service",
-        default: channelOutput.RoleArn,
-      },
-    ];
-    answers = await inquirer.prompt(questions);
+    answers = {
+      FromAddress: await prompter.input(`The 'From' Email address used to send emails`, channelOutput.FromAddress),
```

Comment:
initial values need to be specified like:
```suggestion
      FromAddress: await prompter.input(`The 'From' Email address used to send emails`, {initial: channelOutput.FromAddress}),
```
also, if it's possible to type `channelOutput` this would have been caught by typescript